### DISCUSSION TITLE
do you have any examples that use Orange3 for brain image analysis and network analysis?

### DISCUSSION THREAD

**Discussion by cosmosanalytics** [biolab/orange3#6819] [General]
**What's your use case?**

Use Orange3 for brain image analysis and network analysis

**What's your proposed solution?**

I want to learn how to use Orange3 for brain image analysis and network analysis

**Are there any alternative solutions?**
There are many alternatives, but I like the simplicity of Oranges, and I want to follow examples if someone has already done so

**Comment by janezd** [comment]
I'm moving this from *issues* to *discussions*. I hope that there's anybody who's been doing that and can help you.
